src/applications/vaos/tests/mocks/setup.js

```diff
@@ -306,19 +306,22 @@ export async function setVaccineFacility(store, facilityId, facilityData = {}) {
  * @export
  * @async
  * @param {ReduxStore} store The Redux store to use to render the page
- * @param {string|RegExp} label The string or regex to pass to *ByText query to get
- *   a radio button to click on
+ * @param {string} value The string value of the radio button to click on
  * @returns {string} The url path that was routed to after clicking Continue
  */
-export async function setClinic(store, label) {
+export async function setClinic(store, value) {
   const screen = renderWithStoreAndRouter(
     <Route component={ClinicChoicePage} />,
     {
       store,
     },
   );
 
-  fireEvent.click(await screen.findByLabelText(label));
+  const radioSelector = screen.container.querySelector('va-radio');
+  const changeEvent = new CustomEvent('selected', {
+    detail: { value },
```

Comment:
Similar to the E2E tests, I'm opting to select by value which works more consistently.